README.md

```diff
@@ -189,36 +174,37 @@ about configuring kubectl with the
 
 ### Using Ingress
 
-The kubernetes worker charm supports deploying an NGINX ingress controller.
-In Kubernetes, workloads are declared using pod, service, and ingress definitions.
+The kubernetes-worker charm supports deploying an NGINX ingress controller. 
+Ingress allows access from the Internet to containers inside the cluster 
+running web services. In Kubernetes, workloads are declared using pod, service,
+and ingress definitions.
 
-An ingress controller is provided to you by default, deployed into the [default
-namespace](http://kubernetes.io/docs/user-guide/namespaces/) of the cluster.
-If one is not available, you may deploy this with
+An ingress controller is provided to you by default, deployed into the 
+[default namespace](http://kubernetes.io/docs/user-guide/namespaces/) of the 
+cluster. If one is not available, you may deploy this with:
 
 ```
 juju config kubernetes-worker ingress=true
 ```
 
-Ingress resources are DNS mappings to your containers,
-routed through [endpoints](http://kubernetes.io/docs/user-guide/services/)
-
+Ingress resources are DNS mappings to your containers, routed through
+[endpoints](http://kubernetes.io/docs/user-guide/services/)
 
-As an example for users unfamiliar with kubernetes, we've packaged an action
-to both deploy an example and clean itself up:
+As an example for users unfamiliar with Kubernetes, we packaged an action to 
+both deploy an example and clean itself up:
 
 ```
 juju run-action kubernetes-worker/0 microbot replicas=5
 ```
 
-This performs the following:
+This action performs the following steps:
 
-It creates a deployment titled 'microbots' comprised of 5 replicas defined
+- It creates a deployment titled 'microbots' comprised of 5 replicas defined
 during the run of the action. It also creates a service named 'microbots'
 which binds an 'endpoint', using all 5 of the 'microbots' pods.
 
-Finally, it will create an ingress resource, which points at a [xip.io](https://xip.io)
-domain to aid in simulating having a proper DNS
+- Finally, it will create an ingress resource, which points at a 
```

Comment:
I realize I did this initially, but cold you add a bit about how to clean it up?

```
juju run-action kubernetes-worker/0 microbot delete=true
```